_posts/2017-05-08-refactoring-identities.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "We're changing the way identities work"
+author: Kenton Varda
+authorUrl: https://github.com/kentonv
+---
+
+Over the next few weeks, I'll be making a major change to the way Sandstorm handles user accounts and identities. My goal is to make things far less confusing.
+
+Most Sandstorm users probably have no idea that these features exist, and so won't notice the change. However, if you've linked multiple "identities" (multiple e-mail, Google, or Github accounts) to your Sandstorm account, you may want to read this.
+
+### History: The current system and why it was built this way
+
+Way back in late 2015, we introduced the concept of "multiple identities" in Sandstorm. Ever since, a user account on a Sandstorm server has been able to have multiple identities attached to it, each with a different profile (name, picture, etc.). Once a user has multiple identities, they can choose which identity to act as when using apps. When users share with each other, they often do so "by identity", meaning you choose the identity (not the account) with whom you want to share. It's even possible for multiple accounts to share a common identity, in order to have multiple people "acting as" the same persona.
```

Comment:
Hmm, I'm not sure what you're looking for here. The notion of a "user account" seems self-explanatory to me, but I'm open to suggestions.

I addressed your other comments.